### PR TITLE
readme: Syntax highlight code snippets

### PR DESCRIPTION
**Pull request by ejj**
This patch syntax highlights the JavaScript code snippets in the
README.

### CODE REVIEW

File: docs/readme_test.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	blockStart = "<!-- BEGIN CODE -->\n"
-	blockEnd   = "<!-- END CODE -->\n"
+	blockStart = "```javascript\n"
```
Comment (tejasmanohar) [inline docs/readme_test.go]: maybe match on "```?\n" since we may use other langs in readme?

Comment (ejj) [inline docs/readme_test.go]: No.  This is just for testing the quilt.js code.  Stuff in other languages shouldn't run through the readme test.

**Comment by tejasmanohar** [comment]
small nit but LGTM

File: docs/readme_test.go (hunk 1 above)
Comment (tejasmanohar) [inline docs/readme_test.go]: makes sense

**Pull request merged**